src/apps/TxBuilder/hooks/useServices/interfaceRepository.ts

```diff
@@ -48,7 +48,13 @@ class InterfaceRepository {
     const abi = JSON.parse(abiString);
 
     const methods = abi
-      .filter((e: any) => e.constant === false)
+      .filter((e: any) => {
+        if (["pure", "view"].includes(e.stateMutability)) {
+          return false;
+        }
+
+        return !!e.constant === false;
```

Comment:
If I understand this correctly this means if `e.constant` is `undefined` or `false` then we can use it, right? (my JS is not super good for this kind of magic :P)

Reply:
`!!` is to handle falsy values, `constant` is optional (sometimes could come as undefined), in that case, we should assume it's false.